src/services/messages-service.js

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios';
```

Comment:
Please add JSDOC.

Reply:
Added JSDOC in https://github.com/sabrasisler/FSE-Final-Project-React/pull/7 since my PRs are stacked.